pkgs/stdenv/linux/default.nix

```diff
@@ -624,7 +633,7 @@ in
 
       shell = cc.shell;
 
-      inherit (prevStage.stdenv) fetchurlBoot;
+      inherit fetchurlBoot;
```

Comment:
Why did this need to change?

Reply:
No reason in particular, it was easier to remove all fetchurl/fetchurlBoot instances in stdenv/linux and rewrite it from scratch to when working on this PR, and I haven’t minimized the diff yet because some commits should be split into separate PRs (e.g. openssl stuff).